packages/pwa-kit-create-app/assets/templates/chakra-storefront/config/default.js.hbs

```diff
@@ -0,0 +1,218 @@
+/*
```

Comment:
Bonus: the .hbs version is now correct too... default.js, rather than default.json.